src/main/java/gov/loc/repository/bagit/verify/FileCountAndTotalSizeVistor.java

```diff
@@ -30,6 +31,12 @@ public FileVisitResult preVisitDirectory(final Path dir, final BasicFileAttribut
       logger.debug("Skipping {} cause ignore hidden files/directories", dir);
       return FileVisitResult.SKIP_SUBTREE;
     }
+    //needed because Files.isHidden() doesn't work if the file is a directory
+    if(ignoreHiddenFiles && System.getProperty("os.name").contains("Windows") && 
```

Comment:
It feels like this logic should be refactored into a utility function since we have multiple copies of it. I generally also prefer to wrap compound clauses in parentheses to clarify that the second & third components are paired:

```java
if(ignoreHiddenFiles && (System.getProperty("os.name").contains("Windows") && 
                         Files.readAttributes(dir, DosFileAttributes.class).isHidden())){
```